arabic/dream/dream.md

```diff
@@ -2,4 +2,4 @@
 احلم......الاحلام تتحقق اذا صدقة النية
 ولكن الاحلام لا تتحقق الا بالتوكل على الله والاخذ بالاسباب
 الان انت فى بداية الطريق ماذا ستفعل
-1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md
+1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.